docs/source/markdown/podman-create.1.md

```diff
@@ -1086,15 +1086,25 @@ Set the user namespace mode for the container. It defaults to the **PODMAN_USERN
 
 Valid _mode_ values are:
 
-- **auto[:**_OPTIONS,..._**]**: automatically create a namespace. It is possible to specify these options to `auto`:
+**auto[:_OPTIONS,..._]**: automatically create a unigue user namespace.
+
+The `--userns=auto` flag, requires that the user name `containers` and a range of subordinate user ids that the Podman container is allowed to use be specified in the /etc/subuid and /etc/subgid files.  Example: `containers:2147483647:2147483648`. Podman allocates unigue ranges of UIDs and GIDs from the `containers` group. The size of the ranges is based on the number of UIDs required in the image. The number of UIDs and GIDs can be overridden with the `size` option.
```

Comment:
Should we mention root vs rootless usage here?

Reply:
Added a comment about only being rootless.